docker/Dockerfile.go_generate_dependency

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.7 as go_generate_dependency_builder
+RUN apk add --no-cache build-base git
+
+RUN apk update
+RUN apk add --no-cache go>1.10
+ENV GOPATH=/go \
+    PATH=/go/bin/:$PATH
+RUN go get -u -v \
+    github.com/jteeuwen/go-bindata/... \
```

Comment:
Do we need to peg the version numbers of these as well?

Reply:
Don't think they are actively pushing releases.. go-bindata last release was 2014 and for counterfeiter 2017, thus using from head seems better as we would anyways bind to a good know tried version using the dockerfile
What do you think?